src/main/java/frc/robot/subsystems/gpm/Arm.java

```diff
@@ -279,12 +270,12 @@ public void periodic() {
         motors[0].setControl(m_request.withOutput(dutyCycle));
 
         // report the arm angle in radians
-        SmartDashboard.putNumber("abs value", encoder.getAbsolutePosition());
+        //SmartDashboard.putNumber("abs value", encoder.getAbsolutePosition());
         
```

Comment:
This report is in revolutions. It is needed when the absolute encoder slips, so lI would never comment it out.

Reply:
Value reported below, so this group may be deleted.